nemo/collections/common/data/lhotse/nemo_adapters.py

```diff
@@ -360,6 +361,11 @@ def _iter_sequential(self, tar_path, shard_manifest, manifest_path) -> Generator
                     yield data, raw_audio, tar_info
                 except KeyError as e:
                     if self.skip_missing_manifest_entries:
+                        logging.warning(
```

Comment:
Please revert, the user set a flag to silently omit missing manifest entries, we shouldn't spam them with warnings.

Reply:
done